lib/extractor/index.ts

```diff
@@ -41,7 +41,7 @@ async function getDockerArchiveLayersAndManifest(
 function getContent(
   extractedLayers: ExtractedLayers,
   extractAction: ExtractAction,
-): string | undefined {
+): string | Buffer | undefined {
```

Comment:
@ivanstanev, In which case the return type is of type `Buffer`?

Reply:
@moshikod When we read the RPM DB file content: it is a binary file!